README.md

```diff
@@ -12,10 +12,14 @@ For more information on these inputs, see the [API Documentation](https://develo
 
 - `tag_name`: The name of the tag for this release
 - `release_name`: The name of the release
-- `body`: Text describing the contents of the release
+- `body`: Text describing the contents of the release. Optional, and not needed if using `bodyFromFile`.
+- `bodyFromFile`: A file with contents describing the release. Optional, and not needed if using `body`.
```

Comment:
The existing inputs use snake_case. For consistency, I'd expect body_from_file to do so as well. (though I might suggest `body_file`; `body_from_file` sounds like it would be a boolean to me)

Reply:
Hmm, I see your point about `body_from_file`, but I feel like `body_file` sounds odd. Everything else I can think of sounds boolean-ish now though 😆 . Maybe `body_file_input`? I am not real thrilled with that either though.

Reply:
I wouldn't worry about the name so much, that was just an aside 😸 . I think it'd be fine to just switch to snake_case for now and deal with a rename later if a good one strikes you!

https://github.com/softprops/action-gh-release uses `body_path`, though. (just as a point of reference)

Reply:
FWIW, I merged this PR into my own fork and so far 👍

Reply:
Honestly, `body_path` is pretty good. Worth switching to that?

Glad to hear it is working 👍 . I have this in "production" in a few repos now and we haven't had an issue so far.

Reply:
example: https://github.com/tauri-apps/tauri/blob/2e67d2bc9ff7ba09a71a10e92bfd7d9779edebc2/.github/workflows/release-npm.yml#L70-L81

Reply:
Switched it to `body_path`.